-- This should break and throw an error
local myFunction: (number, string): any = 1